meta/netbeansrelease.json

```diff
@@ -574,7 +589,7 @@
     "master": {
         "position": "50000",
         "ant": "ant_latest",
-        "jdk": "jdk_1.8_latest",
+        "jdk": "jdk_1.11_latest",
```

Comment:
Not `jdk_11_latest`?

Reply:
yep your right.

Fixing release date for 12.6 in the same time..

for the vsnetbeansrelease.json we have some common item? (there is release on netbeans release and another before ? )

Reply:
Yes, as far as I know. Have suggested someone involved joins IDE release team so we can coordinate.

Separate file would stop those polluting our release info, but require a bit of duplication.  There are a few other things, like the previous releases in the IDE includes the VSNetBeans releases.  Leaves us free to rely on only information that's relevant for the IDE in here.

Reply:
yep sounds good. merging this for now